base_layer/core/tests/helpers/nodes.rs

```diff
@@ -105,6 +108,7 @@ pub struct BaseNodeBuilder {
     mempool_config: Option<MempoolConfig>,
     mempool_service_config: Option<MempoolServiceConfig>,
     liveness_service_config: Option<LivenessConfig>,
+    p2p_config: Option<P2pConfig>,
```

Comment:
Not related to this PR, but this struct should really be called "TestBaseNodeBuilder"